kotlinx.fuzz.gradle/src/main/kotlin/kotlinx/fuzz/gradle/junit/MethodTestDescriptor.kt

```diff
@@ -1,22 +1,30 @@
 package kotlinx.fuzz.gradle.junit
 
 import java.lang.reflect.Method
+import kotlinx.fuzz.KFuzzConfig
+import kotlinx.fuzz.methodReproducerPath
+import kotlinx.fuzz.regression.listCrashes
 import org.junit.platform.engine.TestDescriptor
 import org.junit.platform.engine.support.descriptor.AbstractTestDescriptor
 import org.junit.platform.engine.support.descriptor.MethodSource
 
 internal class MethodTestDescriptor(
-    val testMethod: Method, parent: TestDescriptor,
+    val testMethod: Method, parent: TestDescriptor, config: KFuzzConfig,
 ) : AbstractTestDescriptor(
     parent.uniqueId.append("method", testMethod.name),
     displayName(testMethod),
     MethodSource.from(testMethod),
 ) {
     init {
         setParent(parent)
+        if (isRegression()) {
```

Comment:
Right now, almost every interaction with `MethodTestDescriptor` starts with `isRegression()`. This seems to be tricky and will cause bugs in the future.
How about to create a separate `MethodRegressionTestDescriptor`? Then, we can create them in `ClassTestDescriptor.addAllChildren` instead of regular `MethodTestDescriptors` (also we can rename it to `MethodFuzzTestDescriptor`

Reply:
Fixed